src/containers/ProjectDashboardContainer.jsx

```diff
@@ -20,18 +22,57 @@ const propTypes = {
 };
 
 class ProjectDashboardContainer extends Component {
-
+  constructor() {
+    super();
+    this.handleSearch = this.handleSearch.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+    this.state = {
+      searchText: '',
+      translationLanguage: '',
+    };
+  }
   componentDidMount() {
     const { actions } = this.props;
     actions.fetchProject(this.props.params.project_id);
   }
 
+  handleSearch(event) {
+    this.setState({
+      searchText: event.target.value,
+    });
+  }
```

Comment:
Does Panoptes allow you to store multiple translations in the same language for a resource? Sounds like this is a question to be handled by the API.

We could have the javascript look at the resources returned and apply some filtering to choose the correct one. Or we could ask the user to choose between them.